source/documentation/user-journeys.html.md.erb

```diff
@@ -9,73 +9,99 @@ weight: 40
+Any goods not classed as category 1 or 2 are standard goods. Standard goods can benefit from SPIMM with the reduced data requirements. The TGP data model calls standard goods 'category 3'.
 
-    b. If a trader cannot exempt an assessment, it’s recommended that you don’t show any remaining assessments to the user. This is because the outcome of those assessments will not impact the category of the trade.
+#### Categorisation result
 
-10. When the trader has been shown all the applicable category assessments, an overall category for the trade can be
-determined. If the trader was able to exempt all assessments, then the trade is likely to be of standard goods
-(referred to as category 3 in the TGP data model). If the trader was not able to exempt all assessments, then the
-Category will be 1 or 2 (depending on the trade).
+When the trader has been shown all the applicable category assessments, a category for the trade can be determined.
 
-    More information on the categorisation process is available on the [Trade Tariff Categorisation API pages](https://api.trade-tariff.service.gov.uk/green-lanes.html#windsor-framework-green-lanes-categorisation-api).
+If the trader was able to exempt all assessments, then the trade is likely to be of standard goods. If the trader was not able to exempt all assessments, then the category will be 1 or 2 depending on the trade.
 
-11. If the Trade Tariff commodities API returned a measurement unit for the provided commodity code, the trader can optionally populate a supplementary unit quantity for their goods record. This would be recommended if the quantity rarely changes for their goods movements.
+You can find out more information about categorisation on the [Trade Tariff Categorisation API pages](https://api.trade-tariff.service.gov.uk/green-lanes.html).
 
-12. Once all of this information has been captured, your software can use the Create a goods record endpoint to add
-the record to the trader’s TGP.
+### 5\. Measurement unit
 
-    Refer to the [API specification](https://developer.qa.tax.service.gov.uk/api-documentation/docs/api/service/trader-goods-profiles/1.0/oas/page) for more information on this endpoint.
+The Trade Tariff commodities API may return a measurement unit for the provided commodity code. The trader can populate a supplementary unit quantity for their goods record. Traders will benefit from this if the quantity rarely changes on their movements.
+
+### 6\. Creating a goods record
+
+Once the trader enters this information, your software can use the 'Create a goods record' endpoint to add the record to the trader’s TGP.
+
+Refer to the [API specification](https://developer.qa.tax.service.gov.uk/api-documentation/docs/api/service/trader-goods-profiles/1.0/oas/page) for more information on this endpoint.
```

Comment:
https://developer.service.hmrc.gov.uk/api-documentation/docs/api/service/trader-goods-profiles/1.0/oas/page

Need to remove "qa.tax"